.pipelines/prod-release.yml

```diff
@@ -3,25 +3,22 @@ pr: none
 trigger: none
 
 stages:
-  - stage: Deploy_US_Sector
-    displayName: 🚀 Deploy US Sector
+  - stage: Deploy_USSector
+    dependsOn: []
+    displayName: 🚀 Deploy USSector
     jobs:
       - template: ./templates/template-job-deploy-azure-region.yml
         parameters:
           environment: RP-Prod
-          location: eastus
+          locations: 
+            - eastus
```

Comment:
Is the deployment order followed by eastus first, then eastus2?

Reply:
no they deploy in parallel. controlling sector by sector will be done by making a dependency graph at the stage level.

Reply:
here is documentation . If you read down on dependencies you'll find this. "Example jobs that build in parallel (no dependencies):"
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml